content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -102,7 +102,7 @@ different Kubernetes components.
 | `EvenPodsSpread` | `false` | Alpha | 1.16 | 1.17 |
 | `EvenPodsSpread` | `true` | Beta | 1.18 | |
 | `HPAScaleToZero` | `false` | Alpha | 1.16 | |
-| `HugePageStorageMediumSize` | `false` | Alpha | 1.18 | |
```

Comment:
Retain this row, amending the last column to show the last version when it was alpha (ie, 1.18)

Reply:
@sftim thank you for the review! updated.